src/index.ts

```diff
@@ -34,6 +34,12 @@ export {
   RadioGroup,
   RadioGroupProps,
   Stack,
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableRow,
+  TableContainer,
```

Comment:
I would suggest exporting also the related props type ( `TableProps`, `TableBodyProps`, etc..) as for the other components

Reply:
perfect, thx!